controllers/virtualmachineinstance_controller.go

```diff
@@ -18,49 +18,54 @@ package controllers
 
 import (
 	"context"
+
 	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
-	"k8s.io/apimachinery/pkg/runtime"
 	v1 "kubevirt.io/api/core/v1"
-	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // VirtualMachineInstanceReconciler reconciles a VirtualMachineInstance object
 type VirtualMachineInstanceReconciler struct {
 	client.Client
 	Log    logr.Logger
 	Scheme *runtime.Scheme
+	// TODO ZoneManager zoneManager
 }
 
-func (r *VirtualMachineInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	r.Log.Info("In reconcile loop")
-
-	// TODO(user): your logic here
+func (r *VirtualMachineInstanceReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
+	vmi := &v1.VirtualMachineInstance{}
+	err := r.Client.Get(context.TODO(), request.NamespacedName, vmi)
```

Comment:
I think that one time Edy suggested to use `context.Background` instead
It is more deterministic that it is an empty context without place holding for future changes
https://www.mongodb.com/community/forums/t/go-why-context-todo-and-not-context-background/5009